src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx

```diff
@@ -319,38 +319,38 @@ You can use the [Apache log4net version 2.0.8 or higher](https://logging.apache.
+
+      ```xml
+      <log4net>
+       <root>
+         <level value="ALL" />
+         <appender-ref ref="NewRelicAppender" />
+       </root>
+
+       <appender name="NewRelicAppender" type="NewRelic.LogEnrichers.Log4Net.NewRelicAppender, NewRelic.LogEnrichers.Log4Net" >
+         <threshold value="ALL"/>
+         <appender-ref ref="FileAppender" />
+       </appender>
+
+       <appender name="FileAppender" type="log4net.Appender.FileAppender">
+         <file value="C:\logs\log4netExample.log.json" />
+         <param name="AppendToFile" value="true" />
+         <layout type="NewRelic.LogEnrichers.Log4Net.NewRelicLayout, NewRelic.LogEnrichers.Log4Net">
+         </layout>
+       </appender>
+      </log4net>
+      ```
 
-    ```xml
-    <log4net>
-     <root>
-       <level value="ALL" />
-       <appender-ref ref="NewRelicAppender" />
-     </root>
-
-     <appender name="NewRelicAppender" type="NewRelic.LogEnrichers.Log4Net.NewRelicAppender, NewRelic.LogEnrichers.Log4Net" >
-       <threshold value="ALL"/>
-       <appender-ref ref="FileAppender" />
-     </appender>
-
-     <appender name="FileAppender" type="log4net.Appender.FileAppender">
-       <file value="C:\logs\log4netExample.log.json" />
-       <param name="AppendToFile" value="true" />
-       <layout type="NewRelic.LogEnrichers.Log4Net.NewRelicLayout, NewRelic.LogEnrichers.Log4Net">
-       </layout>
-     </appender>
-    </log4net>
-    ```
+      After you configure the log4net extension and update your logging file, you can configure your extension to send data to New Relic. There are [several options](https://docs.newrelic.com/docs/logs/forward-logs/enable-log-management-new-relic/#log-forwarding) for forwarding your logs. Here is an example configuration using the infrastructure agent for logs in context:
 
-    After you configure the log4net extension and update your logging file, you can configure your extension to send data to New Relic. There are [several options](https://docs.newrelic.com/docs/logs/forward-logs/enable-log-management-new-relic/#log-forwarding) for forwarding your logs. Here is an example configuration using the infrastructure agent for logs in context:
+      ```yml
+      logs:
+        - name: application-log
+          file: C:\logs\log4netExample.log.json # Path to a single log file
+      ```
 
-    ```yml
-    logs:
-      - name: application-log
-	      file: C:\logs\log4netExample.log.json # Path to a single log file
-    ```
-    
-    #[Documentation for using the infrastructure agent forwarder](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file)
+      #[Documentation for using the infrastructure agent forwarder](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file)
```

Comment:
I don't think this line should begin with #. Maybe reword to:
See the [infrastructure agent forwarder documentation]/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file).